include/ginkgo/core/base/executor.hpp

```diff
@@ -877,9 +877,9 @@ class CudaExecutor : public detail::ExecutorBase<CudaExecutor>,
     static int get_num_devices();
 
     /**
-     * Get the number of cores per SM of this executor.
+     * Get the number of warps per SM of this executor.
      */
-    int get_num_cores_per_sm() const noexcept { return num_cores_per_sm_; }
+    int get_num_warps_per_sm() const noexcept { return num_warps_per_sm_; }
```

Comment:
Any particular reason why you changed from `num_cores_per_sm` to `num_warps_per_sm` ? Just wondering

Reply:
I change `num_cores_per_sm` to `num_warps_per_sm` because I do not use `num_cores_per_sm` to compute in AMD. I do not find the proper relationship of cores in cuda and amd.
`num_warps_per_sm` is `num_cores_per_sm`/32 in CUDA.

Reply:
Ah, I see.